lib/src/common/repos_service.dart

```diff
@@ -513,28 +515,74 @@ class RepositoriesService extends Service {
         .objects("GET", "/repos/${slug.fullName}/releases", Release.fromJson);
   }
 
-  /// Fetches a single release.
+  /// Fetches a single release by the release ID.
   ///
   /// API docs: https://developer.github.com/v3/repos/releases/#get-a-single-release
-  Future<Release> getRelease(RepositorySlug slug, int id) =>
+  Future<Release> getReleaseById(RepositorySlug slug, int id) =>
```

Comment:
Would you be opposed to just leaving this method name as-is?

Reply:
I renamed it because since there are multiple methods for getting the release, which we will now support, `getRelease` would be ambiguous

Reply:
I fully support the new name `getReleaseById` so let's leave it named that. I'd like the original named method still in there as deprecated and have it just delegate to the new one like:
```
@deprecated 
Future<Release> getRelease(RepositorySlug slug, int id) => getReleaseById(slug, id);
```